src/Broadway/EventStore/InMemoryEventStore.php

```diff
@@ -44,6 +44,29 @@ public function load($id)
     /**
      * {@inheritDoc}
      */
+    public function loadFromPlayhead($id, $playhead)
+    {
+        $id = (string) $id;
+
+        if (!isset($this->events[$id])) {
```

Comment:
Shouldn't this throw an exception? At least when given playhead is bigger then 0?